modular_skyrat/modules/novaya_ert/code/head.dm

```diff
@@ -14,10 +12,48 @@
 		return FALSE
 	return ..()
 
+/obj/item/clothing/head/helmet/nri
+	name = "\improper ShZhP-3 Mk.2 helmet"
+	desc = "A ShZhP-3 helmet, meant to be used in tandem with the PPS-4 armor vest. A modern non-powered body armor designed for use by the imperial militarised police forces commonly widespread across the border, as well as the planetary guard. Alt-click to toggle its ballistic visor; doesn't provide any additional bonuses but it looks cool as hell."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	icon_state = "xenohelm"
+	armor = list(MELEE = 35, BULLET = 45, LASER = 25, ENERGY = 30, BOMB = 45, BIO = 0, FIRE = 50, ACID = 50, WOUND = 25)
+	flags_cover = HEADCOVERSEYES | HEADCOVERSMOUTH
```

Comment:
Do not put HEADCOVERSMOUTH if the helmet does not actually cover the mouth I feel like this should be obvious

Reply:
oops i knew i forgot something